src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -392,5 +394,70 @@ public void deleteProjectInvalidUuidTest() {
         Assert.assertEquals(404, response.getStatus(), 0);
     }
 
+    @Test
+    public void patchProjectTest() {
```

Comment:
Can we split these 4 test cases into separate methods? Makes it easier to spot what went wrong in build logs, but also reduces cognitive complexity. Currently you need to keep track of whether e.g. `p1` is modified and *how* throughout the method.